src/modules/core/components/Button/Button.tsx

```diff
@@ -21,6 +21,7 @@ export interface Appearance {
     | 'ghost'
     | 'underlinedBold'
     | 'blue'
+    | 'blueWithBackground'
```

Comment:
At some point we'll have to refactor all the _theme_ names we use across the dapp and come up with more _clever_ names for all of them.

And have them standardized across all components.

But that's something for another time. _(Maybe when we port them over to the CDapp?)_

Reply:
I raised an issue in cdapp to keep track of it